Formula/carto-grpc-server.rb

```diff
@@ -10,7 +10,7 @@ class CartoGrpcServer < Formula
   depends_on "cmake" => :build
   depends_on "go" => :build
   depends_on "ninja" => :build
-  depends_on "abseil"
+  depends_on "abseil@20220623"
```

Comment:
```suggestion
  depends_on "pkg-config" => :build
  depends_on "abseil@20220623"
```
Fails on linux without pkg-config.